### PR TITLE
V5 docker revised guide, updated community links and resources, added instructions for V6 testnet install for Arch Linux

### PR DESCRIPTION
**Pull request by Valcyclovir**
This is BRX from the community channel reporting in, I am bringing in some changes to Milian's guide on his complete installation guide. This guide is being widely shared among the community and making sure it is updated frequently is crucial ! Thanks.

### CODE REVIEW

**Comment by Valcyclovir** [comment]
Please review the previous changes whenever possible. This guide by Milian is widely used by new node runners and I will try to keep it updated as we go.

**Comment by Valcyclovir** [comment]
Arch Linux guide tested using EndeavourOS

**Review by jlsam — CHANGES_REQUESTED** [review]
There are many FTP programs that support sftp.
Filezilla is probably the most well known, but I suggest it should be mentioned as an example only.

To use just a server name in Files, the user needs to have a properly set up ~/.ssh/config file. That is beyond the scope of this guide. Perhaps a more 'manual' example like sftp://user@ip would be more clear

**Comment by Valcyclovir** [comment]
@jlsam Thanks for the feedback ! Since Filezilla is FOSS, I didn't mind straight up suggesting it but you are right, I changed that part a little. Turns out it's also available for Ubuntu (and other Linux platforms) so it will be the go-to suggestion for anyone struggling to move the file over.

**Comment by Valcyclovir** [comment]
Updates today:
- Added blazegraph support for Arch users to reflect most recent updates
- Removed manual installation guide for dockerless since it's outdated (nohub and forever no longer needed), no reasons not to use calvin's one script install
- Updated suggested way to move graphDB to test node server

**Review by djordjekovac — APPROVED** [review]

File: dkg-v6-upcoming-version/setup-instructions-dockerless.md [hunk 1]
```diff
@@ -48,155 +52,23 @@ apt install git -y && cd /root && git clone https://github.com/OriginTrail/ot-no
...
+**Stop / Start / Restarting your node:**
 
 ```
-forever stop index.js
+systemctl stop otnode
+systemctl start otnode
+systemctl restart otnode
 ```
 
 ****\
```
Comment (NZT48) [inline dkg-v6-upcoming-version/setup-instructions-dockerless.md:72]: Please remove this

**Review by NZT48 — APPROVED** [review]

**Pull request merged**